packages/runtime-py/src/main/kotlin/elide/runtime/plugins/python/Extensions.kt

```diff
@@ -0,0 +1,17 @@
+package elide.runtime.plugins.python
+
+import elide.runtime.core.DelicateElideApi
+import elide.runtime.core.PolyglotContext
+import elide.runtime.core.PolyglotValue
+import elide.runtime.core.evaluate
+
+/**
+ * Execute the given Python [source] code, returning the result. This is equivalent to calling
+ * [PolyglotContext.evaluate] and selecting [Python] as source language.
+ *
+ * @param source The Python source code to be executed.
+ * @return The result of the invocation.
+ */
+@DelicateElideApi public fun PolyglotContext.python(/*language=python*/ source: String): PolyglotValue {
```

Comment:
does `/* language=python*/` work!? that's awesome

Reply:
I might need to verify, I believe it worked with Intellij IDEA

Reply:
we love to see it